package.json

```diff
@@ -34,5 +37,13 @@
     "lint": "eslint .",
     "mocha": "nyc mocha",
     "test": "yarn lint && yarn mocha"
+  },
+  "bugs": {
+    "url": "https://github.com/jwplayer/jwplatform-node/issues"
+  },
+  "homepage": "https://github.com/jwplayer/jwplatform-node#readme",
```

Comment:
```suggestion
  "homepage": "https://github.com/jwplayer/jwplatform-node",
```

Reply:
This was auto done by the `npm` cli, so I'm wonderign if there is a reason